src/main/java/com/minecolonies/colony/permissions/Permissions.java

```diff
@@ -453,7 +453,7 @@ public boolean removePlayer(UUID id)
     @Nullable
     public UUID getOwner()
```

Comment:
If this is something we call often we should consider just saving the owner UUID as a field in this class.